vcd/resource_vcd_independent_disk.go

```diff
@@ -77,9 +120,25 @@ var busSubTypes = map[string]string{
 	"ahci":        "vmware.sata.ahci",
 }
 
+var busSubTypesFromValues = map[string]string{
+	"IDE":              "ide",
+	"buslogic":         "buslogic",
+	"lsilogic":         "lsilogic",
+	"lsilogicsas":      "lsilogicsas",
+	"VirtualSCSI":      "virtualscsi",
+	"vmware.sata.ahci": "ahci",
+}
+
 func resourceVcdIndependentDiskCreate(d *schema.ResourceData, meta interface{}) error {
 	vcdClient := meta.(*VCDClient)
 
+	size, sizeProvided := d.GetOk("size")
+	sizeInBytes, sizeInBytesProvided := d.GetOk("size_in_bytes")
+
+	if !sizeProvided && !sizeInBytesProvided {
+		return fmt.Errorf("size in bytes isn't provided")
```

Comment:
Let's use specific field names in messages, suggestion:
"size_in_bytes isn't provided"

Reply:
fixed